packages/core/src/utilities/_index.scss

```diff
@@ -1,22 +1,45 @@
-@import 'background-color';
-@import 'border';
-@import 'border-radius';
-@import 'color';
-@import 'display-visibility';
-@import 'flexbox';
-@import 'float';
-@import 'font-family';
-@import 'font-size';
-@import 'font-style';
-@import 'font-weight';
-@import 'line-height';
-@import 'margin';
+// Utilities Sass
+@import 'background-color/background-color';
+@import 'border/border';
+@import 'border-radius/border-radius';
+@import 'color/color';
+@import 'display-visibility/display-visibility';
+@import 'flexbox/flexbox';
+@import 'float/float';
+@import 'font-family/font-family';
+@import 'font-size/font-size';
+@import 'font-style/font-style';
+@import 'font-weight/font-weight';
+@import 'line-height/line-height';
+@import 'margin/margin';
 @import 'measure/measure';
-@import 'overflow';
-@import 'padding';
-@import 'text-align';
-@import 'text-decoration';
-@import 'text-transform';
-@import 'truncate';
-@import 'vertical-align';
-@import 'word-break'
+@import 'overflow/overflow';
+@import 'padding/padding';
+@import 'text-align/text-align';
+@import 'text-decoration/text-decoration';
+@import 'text-transform/text-transform';
+@import 'truncate/truncate';
+@import 'vertical-align/vertical-align';
+
```

Comment:
`word-break` file and directory needs to be added as well as separating the doc and styles out